EIPS/eip-1234.md

```diff
@@ -0,0 +1,62 @@
+---
+eip: 1234
+title: Constantinople Difficulty Bomb Delay and Block Reward Adjustment
+author: Afri Schoedon (@5chdn)
+discussions-to: https://github.com/ethereum/EIPs/pull/1234
+type: Standards Track
+category: Core
+status: Draft
+created: 2018-07-19
+replaces: 1227
```

Comment:
1227 doesn't appear to be a valid EIP.

Reply:
Now it is #1235? Or do you want me to remove this?

Reply:
Oh, the build was failing because it wasn't merged yet. It should work now.